src/vm/threads.h

```diff
@@ -975,6 +975,26 @@ EXTERN_C void STDCALL OnHijackWorker(HijackArgs * pArgs);
 class BaseStackGuard;
 #endif
 
+// Note: Layout of this must match similarly named structures in jithelpers.cs.
+// TODO: Drop 'new' prefix
```

Comment:
A nit - please follow your TODO: comment here.

Reply:
`TailCallFrame` is an existing type for the previous helper-based tailcall mechanism. Now that we're keeping the x86 helper-based tailcall mechanism around I don't think we will be able to remove it, so I think the new type's name should be changed to something else, but I'm not sure what. Maybe `PortableTailCallFrame`?

Reply:
That sounds like a good name to me.